docs/resources/job.md

```diff
@@ -1,17 +1,13 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "qovery_job Resource - terraform-provider-qovery"
-subcategory: ""
-description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
----
-
 # qovery_job (Resource)
 
 Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
```

Comment:
description to be improved. the qovery job registry makes no sense

Reply:
Gonna do this on the dedicated PR for job.